ponylang-mode.el

```diff
@@ -104,7 +104,7 @@
 
 (defconst ponylang-keywords
   '("actor"
-    "be" "break"
+    "as" "be" "break"
```

Comment:
Done – also added "not" (as in ``if not windows``) and "ifdef" (as in ``ifdef windows``).